get-started/welcome.qmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "Welcome"
+subtitle: Effortless Python web applications
+
+listing:
+  - id: gallery-highlight
+    template: gallery-highlight.ejs
+    contents: gallery-highlight.yml
+
+editor:
+  markdown:
+    wrap: sentence
+---
+
+Create efficient, reactive, and robust web applications and dashboards.
+
+![Code of example shiny application](assets/code-welcome.png){class="img-shadow"}
+
+```python
+# | standalone: true
+
+from shiny.express import ui, render, input
+import seaborn as sns
+
+df = sns.load_dataset("penguins")
+
+ui.input_slider("bins", "Number of Bins", min=5, max=20, value=15, step=1)
+
+
+@render.plot
+def plot():
+    ax = sns.histplot(data=df, x="body_mass_g", bins=input.bins())
+```
```

Comment:
![image](https://github.com/user-attachments/assets/d4d96119-fd9e-4f72-9fbb-1b850aab42ce)

i envision this section being some kind of animation of an application.

Reply:
I replaced this with something cool 😎